.github/workflows/run_test_suite.yml

```diff
@@ -65,7 +65,6 @@ jobs:
 
           sudo perl -MCPAN -e 'install(Memory::Process)'  # no package in Ubuntu
 
```

Comment:
(just an idea: I think that blank line before could also be dropped)

Reply:
sure...why not?